idd/Energy+.idd.in

```diff
@@ -63383,12 +63383,15 @@ Boiler:Steam,
        \key OtherFuel1
        \key OtherFuel2
   N1 , \field Maximum Operating Pressure
-       \units Kpa
+       \required-field
```

Comment:
So if this is actually a required field, we probably need a meaningful default, otherwise we won't know what to put in there with an 8.7-8.8 transition where the field was blank in 8.7.  @EnergyArchmage @mjwitte?